02_configure_host.sh

```diff
@@ -29,15 +29,15 @@ fi
 export VM_NODES_FILE=${VM_NODES_FILE:-}
 if [ ! -z "${VM_NODES_FILE}" ]; then
   echo "VM_NODES_FILE is no longer supported"
-  echo "Please use NUM_MASTERS, NUM_WORKERS and VM_EXTRADISKS variables instead"
+  echo "Please use NUM_MASTERS, NUM_VM_WORKERS and VM_EXTRADISKS variables instead"
```

Comment:
This should still document the NUM_WORKERS interface I think?

Reply:
As in the ansible-playbook right below the information used is NUM_VM_WORKERS, I assumed that was what was needed there.

Reply:
The existing option is NUM_WORKERS, and unless folks are doing advanced testing e.g scale out there is no need to mention NUM_VM_WORKERS here IMO.

NUM_WORKERS is still the primary interface that determines the number of deployed workers.